common_operations/force_merge.json

```diff
@@ -0,0 +1,23 @@
+{
+    "operation": {
+        "operation-type": "force-merge",
+        "request-timeout": 7200
```

Comment:
Let's also include these params https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/main/big5/test_procedures/common/big5-schedule.json#L35-L37

Reply:
@OVI3D0 Could you add these params to allow users to manage segments?